### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by matthewkennedy5**
now the application works with Docker if you run `docker-compose up --build`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+COPY openduck-py/requirements.txt /openduck-py/
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y uvicorn awscli && \ 
+    apt-get clean
+
+RUN pip install --upgrade pip
+RUN pip install Cython
```
Comment (williamluer) [inline Dockerfile:13]: Can this be added to requirements.txt?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+FROM python:3.11
```
Comment (williamluer) [inline Dockerfile:1]: Does this use the GPU?

Comment (matthewkennedy5) [reply Dockerfile:1]: nope

File: Dockerfile (hunk 1 above)
Comment (matthewkennedy5) [reply Dockerfile:13]: its a linux package not a python package

Comment (matthewkennedy5) [reply Dockerfile:13]: for some reason it only works if its specifically installed here

**Review by williamluer — APPROVED** [review]

**Pull request merged**